### PR TITLE
fedekunze/ 1401 Proposals Index

### LINKED ISSUE #1401: Display proposals index
Scope:
- [x] create proposal overview related getter functions in lcdClient
- [x] create a governance module
- [ ] display proposals
- [x] write tests for module
- [x] write tests for lcdClient

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #1401 

_Description:_

screen shot 2018-10-29 at 1 11 03 pm

❤️ Thank you!

---

* [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
* [x] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

**Comment by fedekunze** [comment]
Note: Most of the files changed are renaming and deletions

File: app/src/renderer/components/governance/LiProposal.vue [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<template lang="pug">
+li.li-proposal
```
Comment (jbibla) [inline app/src/renderer/components/governance/LiProposal.vue]: this shouldn't be an `li` anymore - `table-validators` is a proper html table - should be the same for proposals

Comment (fedekunze) [reply app/src/renderer/components/governance/LiProposal.vue]: oh ok I'll update it

Comment (jbibla) [reply app/src/renderer/components/governance/LiProposal.vue]: if you want, you can open an issue to refactor `table-validator` to be more generalized for proposals as well and leave this how it is for now.

**Pull request merged**